doc/process/npm_management.md

```diff
@@ -0,0 +1,34 @@
+# NPM Management
+
+We have a number of modules under the Node.js Foundation including:
+
+* [node-report](https://github.com/nodejs/node-report)
+* [node-inspect](https://github.com/nodejs/node-inspect)
+* [citgm](https://github.com/nodejs/citgm)
+
+We need to make sure that we have continuity in terms of being able to publish
+and update these modules.
+
+We have decided to manage these modules as follows:
+
+* Create a user called [`nodejs-foundation`][] who we always add as one of the
+  collaborators with admin rights and for which the password is maintained by
+  the build workgroup.
+* We would then add individuals as collaborators who can also publish.
+  Generally, a module push can be done by the additional collaborators.
```

Comment:
`Generally, a module push can be done` -> `A module publish will be done`

As below, if this is used as a backup method to add collaborators for continuity, we should emphasize that it's not to override the autonomy of Node.js Foundation module teams.